build.xml

```diff
@@ -591,6 +591,28 @@
         </javac>
     </target>
 
+    <target name="ci-compile-checked"
```

Comment:
Sorry, I didn't mean to check that in… I usually run that before committing (to stay ahead of Jenkins).